spec/20-http_header_format.md

```diff
@@ -242,11 +242,11 @@ base16(trace-flags) = 00  // not recorded
 
 ## Versioning of `traceparent`
 
-Implementation is opinionated about future version of the specification. Current
+Implementations is opinionated about future version of the specification. Current
```

Comment:
Implementations should be opinionated about future version of the specification. The current

Reply:
nice catch! I did a poor find/replace job, but it caught bigger problem. I believe this sentence should be reworded as it talks about spec being opinionated, not implementation. Fixed.